charts/record-linkage/charts/Chart.yaml

```diff
@@ -2,15 +2,6 @@ apiVersion: v2
 name: record-linkage-chart
 description: A Helm Chart for the DIBBs Record Linkage Service
 type: application
-
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
```

Comment:
These changes seem unrelated - is there a reason the comments were removed?

Reply:
It was failing the linter. I guess I could just make the comments shorter in lines? I figured we weren't really using it.